packages/components/src/sortTabs/SortTab.js

```diff
@@ -0,0 +1,153 @@
+// @flow
+
+import * as React from 'react';
+import { View } from 'react-native';
+import { defaultTokens } from '@kiwicom/orbit-design-tokens';
+import {
+  Text,
+  StyleSheet,
+  Icon,
+  Hoverable,
+} from '@kiwicom/universal-components';
+
+import TouchableWithoutFeedback from '../TouchableWithoutFeedback';
+import type { SortIconTypes, SortTypes } from './SortTabs';
+
+export type Props = {|
+  +label: string,
+  +value: SortTypes,
+  +first: boolean,
```

Comment:
What about use `isFirst, isLast, isActive`?

Reply:
good point boolean value should be always prefixed with `is...`

Reply:
I'm not sure what to do about `hovered`. Maybe is could be `isHovered`?

Reply:
Changed everything to be prefixed with `is...`